features/collection/api/src/main/java/org/opennms/netmgt/collection/api/LatencyCollectionResource.java

```diff
@@ -179,7 +175,7 @@ public String getOwnerName() {
 
     @Override
     public ResourcePath getPath() {
-        if (MonitoringLocationUtils.isDefaultLocationName(m_location)) {
+        if (m_location == null || "Default".equals(m_location)) {
```

Comment:
We could use `org.opennms.core.util.LocationUtils#isDefaultLocationName` instead.